packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -40,6 +41,20 @@ function BlockListAppender( {
 	} else if ( canInsertDefaultBlock ) {
 		// Render the default block appender when renderAppender has not been
 		// provided and the context supports use of the default appender.
+		const isDocumentAppender = ! rootClientId;
+		const isParentSelected = selectedBlockClientId !== rootClientId;
```

Comment:
```suggestion
		const isParentSelected = selectedBlockClientId === rootClientId;
```

Reply:
Ah good catch! Thanks Addie 🙌 I must have reverted it back before committing.